### PR TITLE
refactor: Improve library name and version tag usage

### PR DESCRIPTION
**Pull request by pawanpaudel93**

### CODE REVIEW

File: src/utils.ts [hunk 1]
```diff
@@ -1,156 +1,10 @@
...
-
-  max: Int
-}
-
-enum SortOrder {
-  HEIGHT_ASC
-  HEIGHT_DESC
-}
-`;
+export const appVersionTag = { name: 'ArweaveKit', value: '1.4.9' } as Tag;
```
Comment (7i7o) [inline src/utils.ts:10]: Can we import these 2 from package.json? 🤔 
I guess that would solve problems for tests in the long run. Wdyt?

**Review by 7i7o — APPROVED** [review]
Great work! Left a small comment on utils.ts to improve whenever we can.

File: src/utils.ts (hunk 1 above)
Comment (pawanpaudel93) [reply src/utils.ts:10]: Yes, we could do that but don't know if that will work when the package is used in other projects. I will have to check if it works fine importing from `package.json`

**Review by ropats16 — APPROVED** [review]
All the tests for transaction and graphql functions work as expected in both node and browser environments. The updated alias for `utils` ensures that imports from files other than those in the lib folder work as well across devices.

LGTM!

**Comment by pawanpaudel93** [comment]
Updated a buffer check & added `tsc-esm-fix` package for fixing the issue with relative imports in nodejs.

**Review by 7i7o — APPROVED** [review]
LGTM! 🚀

**Pull request merged**